src/ontology/mondo-edit.obo

```diff
@@ -198956,18 +198956,27 @@ is_a: MONDO:0010108 {source="MONDOLEX:0010274"} ! testicular germ cell tumor
 id: MONDO:0010275
 name: spondyloepimetaphyseal dysplasia, Bieganski type
 def: "Spondyloepimetaphyseal dysplasia, Bieganski type is a rare primary bone dysplasia disorder characterized by infantile-onset, progressive, multiple skeletal deformities in association with slowly progressive central and peripheral neurodegeneration. Patients present short stature, coarse facies, psychomotor regression and cognitive impairment. Imaging shows abnormally-shaped vertebral bodies, small, flat epiphyses, and widened metaphyses, as well as cerebral and cerebellar atrophy and progressive axonal-hypomyelinating neuropathy." [Orphanet:168448]
-subset: ordo_disease {source="Orphanet:168448"}
+subset: ordo_disease {source="Orphanet:168448", source="Orphanet:83629"}
+synonym: "leukoencephalopathy with metaphyseal chondrodysplasia" RELATED [MONDO:Lexical, OMIM:300660]
+synonym: "leukoencephalopathy with metaphyseal chondrodysplasia; LKMCD" RELATED [OMIM:300660]
+synonym: "LKMCD" RELATED [MONDO:Lexical, OMIM:300660]
 synonym: "SEMD X-linked with mental deterioration" RELATED [GARD:0004891]
 synonym: "SEMD, X-linked, with mental deterioration" RELATED [OMIM:300232]
 synonym: "spondyloepimetaphyseal dysplasia x-linked with mental deterioration" RELATED [GARD:0004891]
 synonym: "spondyloepimetaphyseal dysplasia, X-linked, with mental deterioration" RELATED [OMIM:300232]
 xref: GARD:0004891 {source="MONDO:equivalentTo", source="OMIM-shared"}
+xref: ICD10:G11.4 {source="ORDO:83629/attributed", source="ORDO:83629/ntbt", source="MONDO:directSiblingOf", source="Orphanet:83629"}
 xref: ICD10:Q77.7 {source="ORDO:168448/attributed", source="ORDO:168448/ntbt", source="Orphanet:168448"}
 xref: MESH:C536671 {source="MONDO:equivalentTo", source="MONDO:ontobio"}
+xref: MESH:C567065 {source="MONDO:equivalentTo", source="MONDO:ontobio"}
 xref: OMIM:300232 {source="MONDO:equivalentTo", source="Orphanet:168448", source="ORDO:168448/e"}
+xref: OMIM:300660 {source="MONDO:equivalentTo", source="ORDO:83629/e", source="Orphanet:83629"}
 xref: Orphanet:168448 {source="OMIM:300232", source="MONDO:equivalentTo"}
+xref: Orphanet:83629 {source="OMIM:300660", source="MONDO:equivalentTo"}
```

Comment:
Note you are proxy merging two ORDOs (Orphanet:168448, Orphanet:83629). Is this deliberate? Do you want to follow up with ordo to see if they want to retire one of these?

Reply:
OMIM recently moved https://www.omim.org/entry/300660 into omim_300232, hence I merged them.

Yes, I'll follow up with Orphanet and see if they want to do the same.